### PR TITLE
Implement Atomic Design: Buttons

### LINKED ISSUE #185: Implement Atoms and Molecules to App

### PR DESCRIPTION
**Pull request by anhelinakruk**
- Replace the old buttons with the new Atom Button.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "fmt": "prettier --write .",
     "fmt.check": "prettier --check .",
     "lint": "eslint .",
-    "ngrok": "ngrok http http://localhost:5173",
+    "ngrok": "ngrok http --domain=hagfish-fond-honeybee.ngrok-free.app http://localhost:5173",
```
Comment (Uacias) [inline package.json]: Don't push your static domain, possibly the best solution would be to move that to environments

File: src/components/PortfolioValue/PortfolioValue.tsx [hunk 2]
```diff
@@ -304,12 +307,18 @@ export const PortfolioValue = component$<PortfolioValueProps>(
...
+                  size="small"
+                  customClass="absolute left-3/4 top-1/3 !bg-white/10 !px-1"
+                />
+                <Button
+                  variant="iconBox"
+                  leftIcon={<ImgPfButton />}
+                  size="small"
+                  customClass="absolute absolute left-2/4 top-1/3 !bg-white/10 !px-1"
+                />
                 {/* <div class="absolute custom-bg-button opacity-20 h-full left-2/4 right-1/4 "></div> */}
```
Comment (Uacias) [inline src/components/PortfolioValue/PortfolioValue.tsx]: Can we get rid of that commented out part?

**Review by Uacias — CHANGES_REQUESTED** [review]
Overall good, small change requests according to environment variables, and comments.

File: src/components/PortfolioValue/PortfolioValue.tsx (hunk 2 above)
Comment (anhelinakruk) [reply src/components/PortfolioValue/PortfolioValue.tsx]: Do you mean this commented div or customClass?

Comment (anhelinakruk) [reply src/components/PortfolioValue/PortfolioValue.tsx]: @Uacias

**Review by pgajek — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (anhelinakruk) [reply package.json]: Done

File: src/components/PortfolioValue/PortfolioValue.tsx (hunk 2 above)
Comment (anhelinakruk) [reply src/components/PortfolioValue/PortfolioValue.tsx]: Done. Deleted

**Review by pgajek — APPROVED** [review]

**Review by Uacias — APPROVED** [review]
LGTM

**Pull request merged**